can-value.md

```diff
@@ -101,31 +77,21 @@ Use `can-value` when you need an observable that can get or set a property on an
 In the example below, we use [can-value.bind] to get an observable that
 can get _and_ set `outer.inner.key`:
```

Comment:
does this actually work?  What if you set `outer.inner = { key: "abc" }`?

Reply:
It works as expected if you set `outer.inner = { key: "abc" };` (a matching object). It returns an error if `outer.inner` isn't a matching object, like setting it to the number `3` it throws an error. `outer.inner = 3;` errors out when trying to set the `keyObservable` -> `Uncaught TypeError: Cannot create property 'key' on number '3'`

Demo: https://codepen.io/TDillard/pen/VEVGOz?editors=0011